backend/logic/courses/courses.js

```diff
@@ -373,9 +344,8 @@ var assignCourseToCurriculum = async (courseType, courseCode, curriculum) => {
       [curriculum]
     );
   } catch (err) {
-    console.log(err);
     conn.release();
-    throw Error("Internal Server Error!\n");
```

Comment:
I'm using this string to classify 500 (Internal error) , how do you know if it's an internal error if remove it? I checked the `customeResponse` didn't see any 500 handler, did I miss something or ?

Reply:
It defaults to sending back a 500 error code if no error code is specified. If you want to specify a specific code you can do so with: 
```
let error = new Error ("message") 
error.code = 400; 
throw error; 
```